django_project/certification/apps.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import unicode_literals
```

Comment:
In the beginning of a file, please add a docstring like you did in django_project/certification/admin.py